lib/tasks/reset_qa.rake

```diff
@@ -12,6 +12,7 @@ task _reset_qa: %i[
   restore_support_users
   sync_dev_providers_and_open_courses
   generate_test_applications
+  create_persona_users
```

Comment:
Is this enough to get them on QA? Will I need to run this manually somehow?

Reply:
I'm not sure this runs automatically, I reckon prob best to run `create_persona_users` manually